zerver/lib/actions.py

```diff
@@ -2959,7 +2959,11 @@ def internal_send_stream_message(
     message = internal_prep_stream_message(sender, stream, topic, content, email_gateway)
 
     if message is None:
+        error = _("Unable to deliver message to stream {stream}.").format(stream=stream.name)
```

Comment:
I think this should include the actual error message?

Also, I'm a little surprised that this code path would return `None` rather than throw JsonableError, which is where we'd grab the error message from?